com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md

```diff
@@ -109,6 +127,7 @@ The **Lighting** panel has tools that you can use to visualize various component
 | **Clear Shadow Atlas**  | Enable the checkbox to clear the shadow atlas every frame.   |
 | **Range Minimum Value** | Set the minimum shadow value to display in the various shadow debug overlays. |
 | **Range Maximum Value** | Set the maximum shadow value to display in the various shadow debug overlays. |
+| **Log Cached Shadow Atlas Status** | Press the button to log in the console the list of lights placed in the the cached shadow atlas. |
```

Comment:
Press the button to log in to the Console the list of lights placed in the cached shadow atlas.